satellite/mailservice/service.go

```diff
@@ -66,10 +69,39 @@ func New(log *zap.Logger, sender Sender, templatePath string) (*Service, error)
 	//	return nil, err
 	//}
 
-	service.html, err = htmltemplate.ParseGlob(filepath.Join(templatePath, "*.html"))
+	//service.html, err = htmltemplate.ParseGlob(filepath.Join(templatePath, "*.html"))
+	r, err := zip.OpenReader("/tmp/app/assets.zip")
 	if err != nil {
 		return nil, err
 	}
+	for _, f := range r.File {
+		if !strings.HasPrefix(f.Name, "static/emails") {
+			continue
+		}
+		if !strings.HasSuffix(f.Name, ".html") {
+			continue
+		}
+		baseName := filepath.Base(f.Name)
+		fh, err := f.Open()
+		if err != nil {
+			return nil, err
```

Comment:
Is there a better way to do this err chain in this service?

Reply:
no this is fine